service/src/main/java/com/sedliarov/learningtable/model/enums/MessageCode.java

```diff
@@ -13,19 +13,19 @@
 public enum MessageCode {
 
   /**
-   * Error not found linking in message.properties value message.not.found
+   * Error not found linking in messages.properties value error.student.not.found
    */
-  NOT_FOUND("message.not.found"),
+  STUDENT_NOT_FOUND("error.student.not.found"),
```

Comment:
And the same for other
```suggestion
  ERROR_STUDENT_NOT_FOUND("error.student.not.found"),
```

Reply:
Done